devito/core/gpu.py

```diff
@@ -196,14 +198,14 @@ def _specialize_clusters(cls, clusters, **kwargs):
 
         # Reduce flops
```

Comment:
Flops minimization and Reduce flops as comments subsections looks duplicate....
I would keep CSE in the first subtitle, line 49

Reply:
Agreed